Gemfile

```diff
@@ -15,6 +15,7 @@ gem 'pry', '~> 0.10', group: :development, require: false
 gem 'aruba', '~> 0.7.4', require: false
 gem 'rspec', '~> 3.0', require: false
 gem 'cucumber', '~> 2.0', require: false
+gem 'capybara', require: false # middleman-core forces all plugins to declare this
```

Comment:
So this didn't fix it? Or the above is required for backwards compat? https://github.com/middleman/middleman/commit/3ebd902ec3afb3bd72e640bc846df8686eaaddf2